src/main/java/fr/inria/coming/changeminer/analyzer/instancedetector/ChangePatternInstance.java

```diff
@@ -83,11 +83,11 @@ public void setPattern(ChangePatternSpecification pattern) {
 
 	@Override
 	public String toString() {
-		try {
+//		try {
```

Comment:
Why is the try-catch commented? It was added on https://github.com/SpoonLabs/coming/pull/205/commits/48839c17193eba471bac4d96014222e03e2a6c70
(IIf we dont need it, we should remove it instead of commenting it)

Reply:
This try & catch was added (recently) only to execute the tool on BEARS dataset and get the results. But it wasn't a permanent fix.